doc/source/operations/upgrading.rst

```diff
@@ -0,0 +1,726 @@
+=========
+Upgrading
+=========
+
+This section describes how to upgrade from the previous OpenStack release
+series to |current_release|. It is based on the :kayobe-doc:`upstream Kayobe
+documentation <upgrading>` with additional considerations for using StackHPC
+Kayobe Configuration.
+
+Notable changes in the |current_release| Release
```

Comment:
Notable changes in the next release?
One thing I found when updating SKC for Zed was that it was nice and easy to just grep through the repo for "yoga" to find instances of things to edit. Making things like this a variable is very useful when they don't change between releases, but this page in particular will need a rewrite for each release. I think just plaintext writing `zed` here would make more sense.

Reply:
Would it help to have a special release-agnostic comment that you could search for? It seems a shame to hard code things that don't generally need to change.

```
# NOTE(upgrade): Update these notable changes for the current release.
```

Reply:
Sounds good, I'll start adding this in Antelope too.